docs/src/docs/asciidoc/working-with-asciidoctor.adoc

```diff
@@ -70,78 +80,82 @@ This is the equivalent of the following:
 \include::{snippets}/index/http-response.adoc[]
 
 ----
-
+====
 
 [[working-with-asciidoctor-including-snippets-operation-titles]]
-===== Section titles
+===== Section Titles
 
-For each snippet that's including using `operation`, a section with a title will be
-created. Default titles are provided for the built-in snippets:
+For each snippet that includes `operation`, a section with a title is
```

Comment:
I was trying to say something like "For each snippet that is included using the `operation` macro"

Reply:
Ah, so. That works.